packet_packer.go

```diff
@@ -185,9 +191,24 @@ func (p *packetPacker) composeNextPacket(
 	var payloadLength protocol.ByteCount
 	var payloadFrames []frames.Frame
 
+	// STOP_WAITING and ACK will always fit
```

Comment:
Good point, but then ACK frame writing will become much harder anyway (probably with the need to pass in a max len to Write()).